share_extension/android/extension_post/extension_post.js

```diff
@@ -566,6 +579,26 @@ export default class ExtensionPost extends PureComponent {
         );
     };
 
+    renderMessageLengthRemaining = () => {
+        const {value} = this.state;
+        const messageLengthRemaining = MAX_MESSAGE_LENGTH - value.length;
+
+        if (value.length === 0) {
+            return null;
+        }
+
+        let renderStyle = styles.messageLengthRemaining;
+        if (messageLengthRemaining < 0) {
+            renderStyle = {...renderStyle, color: 'red'};
```

Comment:
I don't think this actually works because `styles.messageLengthRemaining` is a number due to how RN StyleSheets work. Instead, you can replace this with an array of styles to pass into the `style` prop like `renderStyle = [styles.messageLengthRemaining, styles.somethingElse];`

Reply:
Of course! You're right, I guess I'd momentarily forgotten that was working in React Native. 

However, the part that bothers me (and perhaps why I didn't catch this during development) is that the code actually works! The style color **does** change on an Android device and here's what I get in a debug console:

Referencing `styles.messageLengthRemaining` gives me an object instead of an expected number.

<img width="730" alt="Screen Shot 2019-09-19 at 23 32 52" src="https://user-images.githubusercontent.com/887849/65295196-fcf2d380-db36-11e9-8503-0554885f10c8.png">

I must be missing something obvious here. Regardless, the implementation is flawed, and I'll fix the dynamic styling change to The Right Way™.

Reply:
Done. Also accounted for themes.

Reply:
Strange, I swear it used a number here. Maybe that got changed at some point?

Reply:
It did use a number earlier. I changed it per your suggestion to use an array to styles in the prop. Better to have something that works as expected, rather than code that inexplicably works 😆 .